beakerx/gradlew

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
yes it can, but then we need to dupicate gradlew libraries for beakerx/beakerx gradle